.github/workflows/wiki.yml

```
@@ -10,6 +10,24 @@ on:
   gollum:
 
 jobs:
+  table_of_contents_check:
+    name: Wiki Check Table of Contents match Headers
+    runs-on: ubuntu-20.04
+    env:
+      CACHE_DIRECTORY: ~/.bazel_cache
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Bazel
+        uses: abhinavsingh/setup-bazel@v3
+        with:
+          version: 4.0.0
+
+      - name: Wiki Check Table of Contents
+        id: wikichecktoc
```

```suggestion
      - name: Check Wiki Table of Contents
        id: checkWikiToc
```

From looking at other IDs in other yml files, the common pattern is to use camel case.

---

Thanks for the insight, @adhiamboperes. I've replaced the ID with camel case in [wiki.yml](https://github.com/oppia/oppia-android/pull/5382/commits/d113747e709ac4755adaedaba40f8d574a8ba356). Upon further exploration, I noticed that [issue_checks.yml](https://github.com/oppia/oppia-android/blob/a0deeea74289c94797dd9d3729ee7c157030ab67/.github/workflows/issue_checks.yml#L25) follows camel case, while [static_check.yml](https://github.com/oppia/oppia-android/blob/a0deeea74289c94797dd9d3729ee7c157030ab67/.github/workflows/static_checks.yml#L110) and most IDs in [unit_tests.yml](https://github.com/oppia/oppia-android/blob/a0deeea74289c94797dd9d3729ee7c157030ab67/.github/workflows/unit_tests.yml#L161) follow snake case. Additionally, one ID in [unit_tests.yml](https://github.com/oppia/oppia-android/blob/a0deeea74289c94797dd9d3729ee7c157030ab67/.github/workflows/unit_tests.yml#L76) follows kebab case.  Should there be any changes made to follow a common convention?

---

It would be better to file a new issue so that we can discuss an implement some standardization.

---

Got it! Will stick with camel case with this [PR](https://github.com/oppia/oppia-android/pull/5382/commits/d113747e709ac4755adaedaba40f8d574a8ba356) then and work on the tests.